src/app/app.component.spec.ts

```diff
@@ -1,32 +1,40 @@
-import { TestBed, async } from '@angular/core/testing';
+import { TestBed, waitForAsync } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { of } from 'rxjs';
 
 import { AppComponent } from './app.component';
+import { LanguageService } from './services/language/language.service';
+import { ExplorerService } from './services/explorer/explorer.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
import { ExplorerService } from "./services/explorer/explorer.service";
```